Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -211,7 +211,7 @@ private func createResolvedPackages(
 
     // Create a map of package builders keyed by the package identity.
     let packageMapByIdentity: [PackageIdentity: ResolvedPackageBuilder] = packageBuilders.spm_createDictionary{
-        let identity = PackageIdentity(url: $0.package.manifest.url)
+        let identity = PackageIdentity(url: $0.package.manifest.packageLocation)
```

Comment:
This is also likely wrong because not every package will necessarily be remote.

Reply:
yup, https://github.com/apple/swift-package-manager/pull/3155/commits/dc768db5dbc32ea8f183c3d725a31877e2736ad7#r552146856